src/python/evaluation/xlsx_run_tool.py

```diff
@@ -0,0 +1,171 @@
+import argparse
+import enum
+import logging.config
+import os
+import re
+import sys
+import traceback
+from pathlib import Path
+from typing import NoReturn, Union
+
+sys.path.append('')
+sys.path.append('../../..')
+
+import pandas as pd
+from openpyxl import Workbook
+from src.python.common.tool_arguments import RunToolArguments
+from src.python.evaluation.common.util import EvaluationProcessNames, script_structure_rule
+from src.python.evaluation.common.xlsx_util import remove_sheet, write_dataframe_to_xlsx_sheet
+from src.python.evaluation.evaluation_config import EvaluationConfig
+from src.python.review.application_config import LanguageVersion
+from src.python.review.common.file_system import create_file, new_temp_dir
+from src.python.review.common.subprocess_runner import run_in_subprocess
+from src.python.review.reviewers.perform_review import OutputFormat
+
+logger = logging.getLogger(__name__)
+
+
+def configure_arguments(parser: argparse.ArgumentParser, run_tool_arguments: enum.EnumMeta) -> NoReturn:
+    parser.add_argument('xlsx_file_path',
+                        type=lambda value: Path(value).absolute(),
+                        help='Local XLSX-file path. '
+                             'Your XLSX-file must include column-names: '
+                             f'"{EvaluationProcessNames.CODE.value}" and '
+                             f'"{EvaluationProcessNames.LANG.value}". Acceptable values for '
+                             f'"{EvaluationProcessNames.LANG.value}" column are: '
+                             f'{LanguageVersion.PYTHON_3.value}, {LanguageVersion.JAVA_8.value}, '
+                             f'{LanguageVersion.JAVA_11.value}, {LanguageVersion.KOTLIN.value}.')
+
+    parser.add_argument('-tool_path', '--tool_path',
+                        default=Path('src/python/review/run_tool.py').absolute(),
+                        type=lambda value: Path(value).absolute(),
+                        help='Path to script to run on files.')
+
+    parser.add_argument('--traceback', '--traceback',
+                        help='If True, column with the full inspector feedback will be added '
+                             'to the output file with results.',
+                        default=False,
+                        type=bool)
```

Comment:
You can use: `action = "store_false"`. https://docs.python.org/3/library/argparse.html#action